### DISCUSSION TITLE
Could you please add Orcid as a login method?

### DISCUSSION THREAD

**Discussion by michi-zuri** [supabase/supabase#4915] [Feature Requests]
Hi

I'd like to publish an app that other researchers can contribute to without signing up to anything, just using the orcid credentials they have for publishing to journals. Could you please add Orcid to the OAuth providers?

https://info.orcid.org/documentation/features/public-api/orcid-as-a-sign-in-option-to-your-system/

All the best and many thanks for the great work!

**Comment by ran-codes** [comment]
upvoting. this would be awesome!
